### ISSUE TITLE
New Version with javascript only

### ISSUE THREAD

**Issue by bryanwillis** [leongersen/noUiSlider#484]
I apologize in advance for wasting your time, because this is more my lack of javascript knowledge than an actual issue.

I used this last year on a backend form for a staffing company and it worked great for me. 

<img width="1062" alt="screen shot 2015-07-18 at 3 19 58 am" src="https://cloud.githubusercontent.com/assets/4042621/8760920/419151ee-2cfc-11e5-915d-055abd1e8bda.png">

However, I'm trying to use it again on a friends <a href="http://disputebills.com">billing site</a> to simply display a number that is 70% the slider number. I got this working no problem <a href="http://codepen.io/candid/details/BNPYbx/">here</a>.

<img width="1141" alt="dispute bill savings" src="https://cloud.githubusercontent.com/assets/4042621/8760955/04e5f6a8-2cfe-11e5-8ca8-a2e1dffccaf5.png">

My problem is that since the new version has changed to pure javascript I'm totally lost. I was trying to use pips, but couldn't get it to work. I figured I' start from scratch and doing the absolute most simple version of noUiSlider possible, but can't for the life of me get it to work now.

```html
<!DOCTYPE html>
<html >
  <head>
    <meta charset="UTF-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<title>NOUI Demo</title>
   <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
    <!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->
   
<link rel='stylesheet' href='http://maxcdn.bootstrapcdn.com/bootstrap/latest/css/bootstrap.min.css'>
<link rel='stylesheet' href='https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css'>
<link rel='stylesheet' href='https://cdnjs.cloudflare.com/ajax/libs/noUiSlider/6.2.0/jquery.nouislider.min.css'>
</head>
<body>

<div id="slider"></div>
    
<script src='https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js'></script>
<script src='https://maxcdn.bootstrapcdn.com/bootstrap/latest/js/bootstrap.min.js'></script>
<script src='https://cdnjs.cloudflare.com/ajax/libs/noUiSlider/6.2.0/jquery.nouislider.min.js'></script>

<script>
 var slider = document.getElementById('slider');
noUiSlider.create(slider, {
	start: [20, 80],
	connect: true,
	range: {
		'min': 0,
		'max': 100
	}
});
</script>
  </body>
</html>
```

Could anybody help point out the obvious here for me? Thanks!

**Comment by leongersen** [comment]
You've included version 6.2, the newest is 8.x. That's probably it ;)

-----Original Message-----

**Comment by bryanwillis** [comment]
Of course!  I was just testing you.... :)

Here is a most basic template with version 8 for anyone else who was confused at first... Correct me if this is wrong @leongersen! Anyway, thanks for the quick help!


http://codepen.io/candid/pen/JdmGxB/

```html
<!DOCTYPE html>
<html>
<head>
<meta charset="UTF-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<title>NoUiSlider 8</title>

<link rel='stylesheet' href='https://cdn.rawgit.com/leongersen/noUiSlider/master/distribute/nouislider.min.css'>
<link rel='stylesheet' href='https://cdn.rawgit.com/Dispute-Bills/wnumb/master/noui-pips.css'>

<style type="text/css">
/* Additional Styling
 *  http://refreshless.com/nouislider/more/#section-styling
 */
.noUi-target {}
.noUi-base {}
.noUi-origin {}
.noUi-handle {}
.noUi-handle-lower {}
.noUi-handle-upper {}
.noUi-connect {}
.noUi-background {}
.noUi-state-tap {}
</style>
</head>
<body>

<div id="slider"></div>

<script src='https://cdn.rawgit.com/Dispute-Bills/wnumb/master/wNumb.js'></script>
<script src='https://cdn.rawgit.com/leongersen/noUiSlider/master/distribute/nouislider.min.js'></script>
<script>
var slider = document.getElementById('slider');
noUiSlider.create(slider, {
    start: [20, 80],
    connect: true,
    range: {
        'min': 0,
        'max': 100
    }
});
</script>
</body>
</html>
```
